tools/setup_helpers/dist_check.py

```diff
@@ -1,18 +1,27 @@
 import os
+import sys
 import platform
 import subprocess
+import glob
+from itertools import chain
 
 from .env import check_env_flag
 
-IS_WINDOWS = (platform.system() == 'Windows')
+IS_WINDOWS = (platform.system() == "Windows")
+IS_CONDA = "conda" in sys.version or "Continuum" in sys.version
```

Comment:
Copypasted from cudnn right? Maybe we should put it in its own file and share it.